### ISSUE TITLE
Cannot open files with FZF

### ISSUE THREAD

**Issue by agusdmb** [junegunn/fzf.vim#629]
- Category
    - [x] Question
    - [ ] Bug
    - [ ] Suggestion
- OS
    - [x] Linux
    - [ ] macOS
    - [ ] Windows
    - [ ] Etc.
- Vim
    - [x] Vim
    - [ ] Neovim

The problem i have **IS** reproducible with a minimal configuration.

So here is the thing, i have debian 9 stable, Neovim "NVIM v0.2.3-1-gd6f9d1df0" (that's the appimage version) i have FZF installed through vim-plug and i use ZSH... FZF is working in zsh, ctrl-t, alt-c, and tab are all working fine. But in vim is another story, the split window pops up with a list full of the file names, when i type in the search works fine, BUT when i press enter to open one (or many) files nothing happens, in fact, the split window goes black, i can hit the Esc key and then move throw the splits and everything works fine but the file doesnt open... (the error is the same if i try :Buffers or :Files)

i found out that, if i hit **ctrl-z**, and then resume by **fg** the files i tryied to open, now suddenly open, and the FZF starts working fine... but only once i do that everytime i open vim... i mean once i do it, i can use FZF inside vim normaly until i close it (not suspend)

this is my **nvim --version**

> NVIM v0.2.3-1-gd6f9d1df0
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/bin/gcc-5 -Wconversion -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -O2 -g -DMIN_LOG_LEVEL=3 -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/home/travis/build/neovim/bot-ci/build/neovim/build/config -I/home/travis/build/neovim/bot-ci/build/neovim/src -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/.deps/usr/include -I/usr/include -I/home/travis/build/neovim/bot-ci/build/neovim/build/src/nvim/auto -I/home/travis/build/neovim/bot-ci/build/neovim/build/include
Compiled by travis@travis-job-a766c230-5e8b-4128-abe6-a12a167eebf5
Features: +acl +iconv +jemalloc +tui 
See ":help feature-compile"
   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/home/travis/build/neovim/bot-ci/build/neovim/build/nvim.AppDir/usr/share/nvim
"
Run :checkhealth for more info

and when the FZF fails, and i quit vim, i can see this errors in the console
> Error detected while processing DirChanged Auto commands for "*":
E475: Invalid argument: Channel doesn't exist
Error detected while processing function deoplete#send_event[1]..deoplete#util#rpcnotify[2]..<SNR>132_notify:
line    9:
E475: Invalid argument: Channel doesn't exist
Error detected while processing DirChanged Auto commands for "*":
E475: Invalid argument: Channel doesn't exist
E475: Invalid argument: Channel doesn't exist
Error detected while processing function deoplete#send_event[1]..deoplete#util#rpcnotify[2]..<SNR>132_notify:
line    9:
E475: Invalid argument: Channel doesn't exist
Error detected while processing DirChanged Auto commands for "*":
E475: Invalid argument: Channel doesn't exist%

**Comment by junegunn** [comment]
> The problem i have IS reproducible with a minimal configuration.

But I can see a name of another plugin, deoplete, which obviously is not included in the minimal configuration, in your error message?

**Comment by agusdmb** [comment]
i just checked commenting out that plugin from my vimrc, and the error message didn't show up this time, but still not working... also, when i added the deoplete plugin back the error didn't show up either :/

i tried the neovim version that is on the debian's repository and that version works fine (which is 0.1.7-4)... i guess either the neovim guys mess up on the appimage build, because is not even 0.2.2, is 0.2.3-1 as i mentioned earlier and.. or there is some changes on this version that breaks (hangs) the fzf window

**Comment by junegunn** [comment]
Closing as a duplicate of #521.
